src/lexer/tokens.rs

```diff
@@ -348,6 +348,9 @@ pub enum Token {
     #[regex("T(IME)?#-?(\\d+(\\.\\d+)?(d|h|ms|m|s|us|ns))+", ignore(case))]
     LiteralTime,
 
+    #[regex("%(B|b|D|d|W|w|X|x)[0-9]+")]
```

Comment:
Norm-grammer says you could use "_" to group the number :-(
I know ... nitpicking ... 
I dont insist on this

Reply:
I'll handle this in #276 As I think I'll adjust the lexer once more to support %X<Identifier>